lib/shopify_api/auth_token_server.ex

```diff
@@ -1,152 +1,88 @@
 defmodule ShopifyAPI.AuthTokenServer do
+  @moduledoc "Write-through cache for AuthToken structs."
+
   use GenServer
-  require Logger
-  alias ShopifyAPI.AuthToken
 
-  @name :shopify_api_auth_token_server
+  alias ShopifyAPI.AuthToken
+  alias ShopifyAPI.Config
 
-  def start_link(_opts) do
-    Logger.info(fn -> "Starting #{__MODULE__} ..." end)
+  @table __MODULE__
 
-    GenServer.start_link(__MODULE__, %{}, name: @name)
+  def all do
+    @table |> :ets.tab2list() |> Map.new()
```

Comment:
🍎 nit pick, consistency. It is multi-line here https://github.com/pixelunion/elixir-shopifyapi/pull/333/files#diff-b9bfc8fd70fa39855bcfa4059f3a007eR11-R14